charts/flyte-binary/templates/crds/flyteworkflow.yaml

```
@@ -0,0 +1,32 @@
+{{- if not .Values.configuration.propeller.createCRDs }}
```

```suggestion
{{- if  .Values.configuration.propeller.createCRDs }}
```
I think this should be the logic?

---

No, the logic is correct. If `propeller` is set to create CRDs at runtime, then we don't want to install them as a part of helm chart installation.

---

You're right. Thanks for your patience, this helped me understand better how the CRD install operation works.